targets/targets.json

```diff
@@ -9410,7 +9412,7 @@
             "PORTIN",
             "PORTINOUT",
             "PORTOUT",
-            "PWMOUT",            
```

Comment:
commit this separately please

Reply:
Hey @0xc0170 did you want the removal of dead space in a separate PR or just a separate commit in this PR?

Reply:
separate commit would work but as there are multiple commits, separate PR is easier.